File: _University/Sem I/FDS 2023/Lab-4/Lab_4_5.py
a = int(input("Enter A Number: "))
b = int(input("Enter A Number: "))
factList = []

for i in range(1, max(a, b)):
    if a % i == 0 and b % i == 0:
        factList.append(i)

print(f"HCF: {max(factList)}")

File: _University/Semester-I/C++ OOPS 2023/Lab-4/Lab_4_5.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    long long fact = 1;
    cout<<"Enter A Number: ";
    cin >> n;

    for (int i = 1; i<=n; ++i){
        fact = fact*i;
    }
    cout<<fact;
}